eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/Imp.hs

```diff
@@ -36,6 +38,7 @@ spec ::
   , InjectRuleFailure "LEDGER" AlonzoUtxoPredFailure era
   , InjectRuleFailure "LEDGER" AlonzoUtxosPredFailure era
   , InjectRuleFailure "LEDGER" AlonzoUtxowPredFailure era
+  , EraPlutusTxInfo L.PlutusV1 era
```

Comment:
Move this constraint to `AlonzoEraTest` instead, this will lead to less constraints in the `spec` for all eras